dependabot is not able to set auto merge on PRs

```
Run gh pr merge --auto --merge "$PR_URL"
  gh pr merge --auto --merge "$PR_URL"
  shell: /usr/bin/bash -e {0}
  env:
    PR_URL: https://github.com/thesuavehog/aws-sns-to-google-chat-space/pull/1[3](https://github.com/thesuavehog/aws-sns-to-google-chat-space/actions/runs/13019679810/job/36317257271#step:4:3)0
    GH_TOKEN: ***
GraphQL: Resource not accessible by integration (enablePullRequestAutoMerge)
Error: Process completed with exit code 1.
```

--- a/.github/workflows/dependabot-auto-merge-pr.yml
+++ b/.github/workflows/dependabot-auto-merge-pr.yml
@@ -6,6 +6,7 @@ on:
 
 permissions:
   pull-requests: write
+  contents: write
 
 jobs:
   dependabot:

